config/core/test-configs.yaml

```diff
@@ -2180,6 +2187,7 @@ test_configs:
   - device_type: meson-g12b-odroid-n2
     test_plans:
       - baseline
+      - kselftest-arm64
```

Comment:
@montjoie Please confirm it's OK to enable this and baseline-nfs in BayLibre's lab.

Reply:
Same here...

Reply:
Looks like some potentially transient infra thing...